CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.2.0
+
+- Return non-zero exit code from executable when incorrect args are used
```

Comment:
Sorry, this is a [nit] but this should be a patch release since we're treating it as a bug fix

Reply:
Updated